tests/misra/suppressions.txt

```diff
@@ -9,4 +9,8 @@ misra-c2012-19.2
 # Advisory: The # and ## preprocessor operators should not be used
 misra-c2012-20.10
 # Required: it's ok re-defining potentially reserved Macro names. Not likely to cause confusion
-misra-c2012-21.1
\ No newline at end of file
+misra-c2012-21.1
+# MUST BE FIXED, mostly in the safety code
+misra-c2012-10.4
```

Comment:
can we fix these instead of ignoring them?

Reply:
Will fix in follow up PR as those violations were always there but were hidden from MISRA by volatile type. If I am going to fix this in this PR - will add ~150-170 lines of  code. And can't fix before as it is hidden.
Also all those violations are in safety code.